.vscode/settings.json

```diff
@@ -12,7 +12,6 @@
       "autoFix": true
     }
   ],
-  "editor.formatOnSave": true,
```

Comment:
If you are going to remove this, you might as we remove the whole file.

Some good reads before doing so:
https://glebbahmutov.com/blog/configure-prettier-in-vscode/
https://levelup.gitconnected.com/setting-up-eslint-with-prettier-typescript-and-visual-studio-code-d113bbec9857

Reply:
After chatting, we've decided that since our linting rules no longer conflict, we will keep `formatOnSave` functionality